SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -39,6 +39,8 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 		return false;
 	}
 
+	checkf(!GetReplicationDriver(), TEXT("Replication Driver not supported, please remove it from config"));
```

Comment:
Worth mentioning here that we'll be looking at this in future, and linking against UNR-318